action.yml

```
@@ -36,5 +48,18 @@ runs:
             args+=(--charts-repo-url "${{ inputs.charts_repo_url }}")
         fi
 
+        if [[ -z "${{ inputs.install_dir }}" ]]; then
+          install="$RUNNER_TOOL_CACHE/cr/${{ inputs.version }}/$(uname -m)"
+          echo $install >> $GITHUB_PATH
```

Always quote variables
```suggestion
          echo $install >> "$GITHUB_PATH"
```